geomesa-core/pom.xml

```diff
@@ -103,6 +103,10 @@
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.scalaz</groupId>
+            <artifactId>scalaz-core_2.10</artifactId>
+        </dependency>
```

Comment:
Maybe.  I talked to David when he was working on the original work, and I tried to figure out if he really needed scalaz.  I don't think it is strictly required.  (See my other comments.)